### DISCUSSION TITLE
Host alpine.js alternative to unpkg

### DISCUSSION THREAD

**Discussion by jamesli2021** [alpinejs/alpine#3235] [4. General]
The CDN is currently having difficult serving Alpine.js and other libraries hosted on unpky, and it's not the first time and has been disruptive due to Cloudflare's issues.

I suggest Alpine.js should host on another CDN site.

**Comment by SimoTod** [comment]
Alpine publish to npm. All CDNs supporting npm will just work.
For example:
https://cdn.jsdelivr.net/npm/alpinejs
https://cdn.jsdelivr.net/npm/@alpinejs/intersect
etc
